iothub/device/src/PayloadCollection.cs

```diff
@@ -107,13 +107,11 @@ public bool Contains(string key)
         /// <summary>
         /// Gets the value of the object from the collection.
         /// </summary>
-        /// <remarks>
-        /// This class is used for both sending and receiving properties for the device.
-        /// </remarks>
         /// <typeparam name="T">The type to cast the object to.</typeparam>
         /// <param name="key">The key of the property to get.</param>
-        /// <param name="value">The value of the object from the collection.</param>
-        /// <returns>True if the collection contains an element with the specified key; otherwise, it returns false.</returns>
+        /// <param name="value">When this method returns successfully, this contains the value of the object from the collection.
```

Comment:
true/false instead of successfully/unsuccessfully?

Reply:
I've used the same language as used in Dictionary.TryGetValue: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue?view=net-5.0